platform/windows/display_server_windows.cpp

```diff
@@ -5851,7 +5853,7 @@ void DisplayServerWindows::_process_key_events() {
 						if (!keysym.is_empty()) {
 							char32_t unicode_value = keysym[0];
 							// For printable ASCII characters (0x20-0x7E), override the original keycode with the character value.
-							if (Key::SPACE <= (Key)unicode_value && (Key)unicode_value <= Key::ASCIITILDE) {
+							if (is_oem && Key::SPACE <= (Key)unicode_value && (Key)unicode_value <= Key::ASCIITILDE) {
```

Comment:
Since it's already restricted to OEM keys, I think ASCII range restriction may not be necessary.

Reply:
Unlikely it will be ever outside this range, but there's no exact list of the valid values OEM keys can produce, and `keycode` still expected to be limited to 102-key US QWERTY keyboard values.

Reply:
Sure I think it would be better to keep the ASCII range restriction for safety.